.github/workflows/get-gke-credentials-it.yml

```diff
@@ -111,3 +115,37 @@ jobs:
             echo "Expected no pods. Got ${pods}"
             exit 1
         fi
+
+  wif:
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    if: ${{ github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]' }}
+    name: with wif
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: google-github-actions/auth@main
+      with:
+        workload_identity_provider: ${{ secrets.WIF_PROVIDER_NAME }}
+        service_account: ${{ secrets.GET_GKE_CRED_SA_EMAIL }}
+        create_credentials_file: true
```

Comment:
This is the default, FWIW. It's probably fine to leave in here though

Reply:
removed